site-content/source/modules/ROOT/pages/blog.adoc

```diff
@@ -8,6 +8,31 @@ NOTES FOR CONTENT CREATORS
 - Replace post tile, date, description and link to you post.
 ////
 
+//start card
+[openblock,card shadow relative test]
+----
+[openblock,card-header]
+------
+[discrete]
+=== Apache Cassandra 4.1 Features: Guardrails Framework
+[discrete]
+==== May 12, 2022
+------
+[openblock,card-content]
+------
+Apache Cassandra 4.1.0 introduces a new framework called Guardrails. This framework helps enforce good practices to avoid poor cluster performance and availability because of certain user actions.
```

Comment:
@dtopdontstop each card can only hold a maximum of 175 characters (or roughly 25 words) so I've edited the summary to make it fit within the card:
```suggestion
Cassandra 4.1 introduces Guardrails - a framework that helps enforce good practices to avoid poor cluster performance and availability because of certain user actions.
```